.github/workflows/ci.yml

```diff
@@ -29,18 +29,19 @@ jobs:
       - name: Install dependencies
         run: yarn install --frozen-lockfile
 
-      - name: Build app
+      - name: Build test-app
         run: yarn build:test
+        working-directory: test-app
 
-      - name: Upload app
+      - name: Upload test-app
         uses: actions/upload-artifact@v3
         with:
           name: dist
-          path: dist
+          path: test-app/dist
```

Comment:
The [migration guide from Embroider](https://github.com/embroider-build/embroider/blob/v1.8.3/PORTING-ADDONS-TO-V2.md) mentioned, we could use [`working-directory`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_iddefaultsrun) to target a specific package.

However, there's a fine print to be aware of:

> You can provide [...] working-directory options for all run steps _**in a job**_. (from GitHub's documentation)

Actions such as `upload-artifact` do not (have to) support `working-directory`. See https://github.com/actions/upload-artifact/issues/196#issuecomment-834291140.